README.md

```diff
@@ -139,6 +139,36 @@ $ mrq-worker fetches
 
 You can interrupt the worker with Ctrl-C once it is finished.
 
+### Run it via the built-in scheduler
+
+Create a `schedule.py` file with the following code:
+
+```
+SCHEDULER_TASKS = [
+    {
+        "path": "fetches tasks.Fetch",
+        "params": {
+            "url": "http://www.google.com"
+        },
+        "queue": "fetch_queue",
+        "interval": 5 # Run every 5 seconds
+    }
+]
+
+SCHEDULER_INTERVAL = 0.1
+```
+Run the schedule using:
+
+```
+mrq-run --scheduler --config schedule.py
```

Comment:
Here you should run a scheduler worker with `mrq-worker --scheduler fetch_queue` . This will spawn a [scheduler greenlet](https://github.com/pricingassistant/mrq/blob/master/mrq/worker.py#L141 that will run the scheduler every SCHEDULER_INTERVAL s)